frontend/src/Editor/Editor.jsx

```diff
@@ -1814,7 +1838,7 @@ class EditorComponent extends React.Component {
     } = this.state;
 
     const appVersionPreviewLink = editingVersion
-      ? `/applications/${app.id}/versions/${editingVersion.id}/${this.state.currentState.page.handle}`
+      ? `/ applications / ${app.id} /versions/${editingVersion.id} /${this.state.currentState.page.handle}`
```

Comment:
I think you have issues with your lint. So please check your lint configuration 

These white spaces will cause problems while opening the app preview page.

Reply:
i have sorted this now. Should be fine. Thanks